Assets/Scripts/PlayerCharacter.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerCharacter : MonoBehaviour {
+
```

Comment:
this script is much better and much cleaner than your others. get rid of some extra space